src/backend/gporca/libgpopt/include/gpopt/mdcache/CMDCache.h

```diff
@@ -44,6 +44,14 @@ class CMDCache
 	// the maximum size of the cache
 	static ULLONG m_ullCacheQuota;
 
+	// if we have cached a relation that contain an index that cannot be used in
+	// the current transaction, then after it ends, the cache must be reset.
+	// to read this relation again.
+	//
+	// here we save the transaction id in which this happened.
+	// for more info see src/backend/access/heap/README.HOT
```

Comment:
maybe this?
```
    // if we have cached a relation without an index, because that index cannot
    // be used in the current transaction (for more info see src/backend/access/heap/README.HOT).
    // We save the transaction's xmin so if later TransactionXmin changes from
    // the saved value, the cache will be reset and the relation will be
    // reloaded with the index usage.
```

Reply:
Maybe that way:
```
  // if we have cached a relation without an index, because that index cannot
  // be used in the current transaction (for more info see src/backend/access/heap/README.HOT),
  // we save the transaction's xmin. If later TransactionXmin changes from
  // the saved value, the cache will be reset and the relation will be
  // reloaded with the index usage.
```